_pytest/python.py

```diff
@@ -43,6 +44,19 @@ def _format_args(func):
     def _format_args(func):
         return inspect.formatargspec(*inspect.getargspec(func))
 
+class _GotAllAttrs():
```

Comment:
can we have a python version specific constant

```
CLASS_TYPES = ...
```
and just always implement isclass with a note to drop the implementation after python2.6 is dropped?